### DISCUSSION TITLE
Import of all postman data

### DISCUSSION THREAD

**Discussion by muhammedfurkan** [Kong/insomnia#7941] [Ideas] [answered]
Hello, I have an idea for those who will switch from Postman to Insomnia. 

All postman data is exported via Postman. By importing all the export data received on a single Insomnia, all collections will come to Insomnia with multiple imports by once click.

Adding such a feature can switch to using Insomnia instead of using Postman.

**Comment by subnetmarco** [comment]
I believe we just enabled bulk import of Postman archives in the latest v10, can you update and try again? @muhammedfurkan

**Reply by muhammedfurkan** [reply]
I updated to the latest version and tried it, it still allows only one import.

**Reply by subnetmarco** [reply]
@yaoweiprc to take a look, because I see a PR that enabled it recently https://github.com/Kong/insomnia/pull/7834

**Comment by yaoweiprc** [comment, marked as answer]
We implemented importing Postman data dump file in #7834 , Please refer to Postman's doc to export a data dump file first.
https://learning.postman.com/docs/getting-started/importing-and-exporting/exporting-data/#export-data-dumps
Postman data dump file contains almost all your collections and environments in Postman, but there are some constraints need to notice:

1. Collections from Postman data dump will be created as new collections in current project with a top level folder. Just like what we do when import an individual collection json file from Postman.
2. Environments from Postman data dump will be created as new global environments in current project.
3. Not include global environments from postman because it's not included in data dump.
4. The data dump does not preserve which workspaces in Postman exist, so we just import all Postman collections and envs to current project in Insomnia.
5. Import data dump in the Insomnia Collection level is not allowed, you can just import in project level.

**Reply by muhammedfurkan** [reply]
I export all the data from Postman. I cannot import all the data on Insomnia at once.
